whois-commons/src/test/java/net/ripe/db/whois/common/conversion/PasswordFilterTest.java

```diff
@@ -45,6 +45,36 @@ public void testFilterPasswordsInMessage() {
                 "delete: adsf\n"));
     }
 
+    @Test
+    public void testFilterBasicAuthHeaderInMessage() {
+        final String input = "" +
+                "Header: Authorization=Basic dDZsUlpndk9GSXBoamlHd3RDR3VMd3F3OjJDVEdQeDVhbFVFVzRwa1Rrd2FRdGRPNg==\n" +
+                "blue: asdfasdfasdf\n" +
+                "yellow%3A++asdfasdfasdf\n" +
+                "green: asdfasdfasdf # password: test\n" +
+                "purple: password\n" +
+                "password:   test1 \n" +
+                "password:test2\n" +
+                "password: test3\n" +
+                "password%3A++test4\n" +
+                "password%3A++test5\n" +
+                "delete: adsf\n";
+
+        assertThat(PasswordFilter.filterPasswordsInContents(input), containsString("" +
+                "Header: Authorization=Basic  FILTERED\n" +
```

Comment:
There are *two* spaces in the output, but the regex doesn't appear to include any spaces in the matching group?

Reply:
removed one space